README.md

```diff
@@ -70,6 +183,61 @@ Here's the `settings.json` file we are using for configuring the vscode:
   "python.defaultInterpreterPath": "/Users/polywrap/Library/Caches/pypoetry/virtualenvs/polywrap-client-abcdef-py3.10"
 }
 ```
-You need to put your virtual env path you got from the poetry under: `python.defaultInterpreterPath`
 
-Once you configure this you should be good to go.
+Keep in mind that these venv paths will vary for each module you run `poetry shell` on. Once you configure these `setting.json` files correctly on each module you should be good to go!
+
+
+# What WASM wrappers can you execute today?
+
+Check these resources to browse a variety available wrappers, for DeFi, decentralised storage, and other development utilites:
+
+- [Wrappers.io](https://wrappers.io/)
+- [Polywrap Integrations Repository](https://github.com/polywrap/integrations)
+
+# Example call
+
+Calling a function of a wrapper from the python client is as simple as creating a file in the `TODO (?polywrap-client)` directory, importing the Polywrap Python Client, calling the Uri where the WASM wrapper is hosted, and specifying any required arguments.
+
+```python
+# get_eth_txns.py
+from polywrap_client import PolywrapClient
+from polywrap_core import Uri, InvokerOptions
+
+async def get_eth_transactions(accountAddress):
+    client = PolywrapClient()
+    #uri = Uri("wrap://ens/defiwrapper.polywrap.eth")
+    uri = Uri(f'fs/{Path(__file__).parent.joinpath("cases", "wrap.wasm").absolute()}')
+    args = {
+        "accountAddress": "'0x123EtherumAddress12312'"
+    }
+    options = InvokerOptions(uri=uri, method="simpleMethod", args=args, encode_result=False)
+    result = await client.invoke(options)
+    return result.result
+
+if __name__ == "__main__":
+    return get_eth_transactions(portfolio_address)
+```
+
+
+## Creating your own tests
+
+TODO: It is suggested to follow a [TDD](https://en.wikipedia.org/wiki/Test-driven_development) approach to build your own implementations. 
```

Comment:
We aren't following TDD so it's not right to recommend it. You can describe other things that you see would be helpful to contributor writing test here though

Reply:
updated